qiskit_dynamics/signals/signals.py

```diff
@@ -299,38 +286,25 @@ def __init__(
         """
         self._dt = dt
 
-        samples = Array(samples)
+        samples = unp.asarray(samples)
 
         if len(samples) == 0:
-            zero_pad = np.array([0])
+            zero_pad = unp.asarray([0])
         else:
-            zero_pad = np.expand_dims(np.zeros_like(Array(samples[0])), 0)
-        self._padded_samples = np.append(samples, zero_pad, axis=0)
+            zero_pad = np.expand_dims(np.zeros_like(unp.asarray(samples[0])), 0)
```

Comment:
Should this line `unp`? I think for safety it may make sense to change it to `unp`.

Reply:
modified at https://github.com/Qiskit/qiskit-dynamics/pull/208/commits/10742b902c76863ba805c983c4dca91db22afd81.